packages/admin/admin/src/common/Loading.tsx

```diff
@@ -58,3 +60,10 @@ const LoadingContainer = styled("div")<Required<Pick<LoadingProps, "behavior">>>
             justify-content: center;
         `}
 `;
+
+const StyledBallTriangle = styled(BallTriangle, {
+    shouldForwardProp: (propName) => propName !== "size",
+})<{ size: number }>`
+    width: ${({ size }) => `${size}px`};
+    height: ${({ size }) => `${size}px`};
+`;
```

Comment:
BallTriangle is a SvgIcon component. I think we should use the recommended way for changing the size of the icon: https://mui.com/material-ui/icons/#size (i.e. remove the `size` prop and styled wrapper and use `fontSize` accordingly).

Reply:
Okay, should be alright now